### PR TITLE
leech: add new package

### PR DESCRIPTION
**Pull request by leonghui**
Author: https://bitbucket.org/alekseyt/
Maintainer: me
Compile tested: mvebu wrt1900acv2 (master)
Run tested: mvebu wrt1900acv2 (master), leech-aria recipe + cron + feed with magnet links

Description:
A set of scripts that parse RSS feeds and downloads files for you.
Recipes are available to send download links to transmission and aria2.

For documentation: https://bitbucket.org/alekseyt/leech/src/master/README.md
For package history: https://forum.archive.openwrt.org/viewtopic.php?id=31294

Signed-off-by: Leong Hui Wong <wong.leonghui@gmail.com>

### CODE REVIEW

File: net/leech/Makefile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+PKG_SOURCE_URL:=https://bitbucket.org/alekseyt/leech.git
+PKG_SOURCE_DATE:=2019-07-17
+PKG_SOURCE_VERSION:=2378e7348a8ac8ed23b6d29be81dc3a73080d920
+PKG_MIRROR_HASH:=04ca1d1f2cf614edf871360c1adbc758d9b85b67296af438449367c068ce5320
+
+PKG_MAINTAINER:=Leong Hui Wong <wong.leonghui@gmail.com>
+PKG_LICENSE:=blessing
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
```
Comment (neheb) [inline net/leech/Makefile]: This can probably be removed if Build/Install is to be empty.

File: net/leech/Makefile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+define Package/leech/description
+  leech downloads files for you if you give it RSS-feeds of your favorite
+  trackers. Configuration files are under /etc/leech.
+endef
+
+define Build/Compile
+endef
+
+define Build/Install
```
Comment (neheb) [inline net/leech/Makefile]: Same as above.

File: net/leech/Makefile (hunk 1 above)
Comment (leonghui) [reply net/leech/Makefile]: Ok, done.

File: net/leech/Makefile (hunk 2 above)
Comment (leonghui) [reply net/leech/Makefile]: And done.

**Comment by BKPepe** [comment]
Those commits should be squashed before merging!

**Comment by leonghui** [comment]
Noted, thanks for your review and merging 👍

**Pull request merged**